jvm/src/main/kotlin/app/cash/trifle/delegate/CertificateAuthorityDelegate.kt

```diff
@@ -22,9 +22,11 @@ interface CertificateAuthorityDelegate {
    *
    * @param issuerCertificate trifle certificate associated with the signer of this cert.
    * @param certificateRequest certificate request used to generate a new certificate
+   * @param validity of the certificate before it expires (days)
    */
   fun signCertificate(
     issuerCertificate: Certificate,
     certificateRequest: CertificateRequest,
+    validity: Int  = CertificateRequest.MOBILE_CERTIFICATE_VALIDITY_PERIOD_DAYS,
```

Comment:
lets move `CertificateRequest.MOBILE_CERTIFICATE_VALIDITY_PERIOD_DAYS` into this class instead and remove it from `CertificateRequest`

Reply:
fixed.